sycl/doc/EnvironmentVariables.md

```diff
@@ -231,6 +231,18 @@ variables in production code.</span>
 | after_addHostAcc | print graph after addHostAccessor method |
 | always | print graph before and after each of the above methods |
 
+
+### `SYCL_UR_TRACE` Options
+
+`SYCL_UR_TRACE` accepts a bit-mask. Supported tracing levels are in the table below
+
+| Option | Description |
+| ------ | ----------- |
+| 1 | Enable basic tracing, which is tracing of UR adapters/devices discovery |
+| 2 | Enable tracing of the UR calls |
+| -1 | Enable all levels of tracing |
```

Comment:
I don't understand the difference between `2` and `-1`

Reply:
`2` enables just the tracing of UR calls, but `-1` enables tracing of the UR calls as well as the basic device/adapter discovery tracing, i.e. `1` and `2` combined (and implicitly also any future tracing levels that may be added).

Reply:
think we can add that explanation to the doc?